DataManagerPlugin.cs

```diff
@@ -1,13 +1,167 @@
-using BepInEx;
+using Bep = BepInEx;
```

Comment:
I think we should generally be avoiding the shortenings unless there are naming collisions, e.g. UObject for UnityEngine.Object

Reply:
The main reason I use these is not to avoid collisions, but for readability - it lets me know at a glance where all the imported names come from. Admittedly, some libraries, like Harmony, BepInEx (for the most part) and Newtonsoft.Json make this superfluous because all their classes have a prefix, so I wouldn't mind yeeting these particular aliases.